README.md

```diff
@@ -401,88 +405,6 @@ username: admin
 password: admin
 ```
 
-Enabling the network topology module
```

Comment:
@pandafy Was there a reason why the following was repeated? I have added just one line above, I think that should do the job, \imo repeating the same setup steps isn't very dry and causes confusion! :smile:

Reply:
These sections are linked from other modules (topology and fw upgrader), they're here so we can quickly show to users interested in particular modules how to deploy them. I would not touch this section now and I would maintain consistency.

Modifying this section will need to update the links that point here and it would be better to deal with it in a different effort, making sure openwisp-radius works should be the priority now.